src/cmds/orgs:create.ts

```diff
@@ -34,7 +34,8 @@ export async function handler(argv: {
     getLoggingPath();
     const { file, noDuplicateNames, includeExistingOrgsInOutput } = argv;
     debug('ℹ️  Options: ' + JSON.stringify(argv));
-    const res = await createOrgs(file, {
+    const filepath = getImportProjectsFile(file)
```

Comment:
The import file is not the file used to create orgs, the file needed here is one created in a `orgs:data` command https://github.com/snyk-tech-services/snyk-api-import/blob/master/docs/orgs.md